spec/factories/snps.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :snp do
+    sequence(:name) { |i| "rs#{i}" }
+    sequence(:position) { |i| i }
+    sequence(:chromosome) { |i| i }
+    genotype_frequency("AA" => 1)
+    allele_frequency("A" => 0, "T" => 0, "G" => 0, "C" => 0)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.